base/stream.jl

```diff
@@ -1063,10 +1063,13 @@ function readuntil(x::LibuvStream, c::UInt8; keep::Bool=false)
     return bytes
 end
 
-uv_write(s::LibuvStream, p::Vector{UInt8}) = GC.@preserve p uv_write(s, pointer(p), UInt(sizeof(p)))
+uv_write(s::LibuvStream, p::Vector{UInt8}) = GC.@preserve p uv_write(s, pointer(p), sizeof(p) % UInt64) # UInt64 is safe here, also for 32-bit, then get UInt32 back from uv_write
```

Comment:
The function you are calling expects a `UInt`. As such I don't see how converting to a `UInt64` here will do any good, and in particular, in how far it would even be "faster"?!

Reply:
UInt is UInt64 on 64-bit, so ok. UInt is Int32 on 32-bit why I comment that this is still ok.

Reply:
I see this has a bug on 32-bit after all, will fix.